etl/steps/data/garden/health/2022-12-28/karlinsky.meta.yml

```diff
@@ -0,0 +1,50 @@
+dataset:
+  sources:
+  #   -
+  #     name: Maddison Project Database 2020 (Bolt and van Zanden, 2020)
+  #     published_by: Bolt, Jutta and Jan Luiten van Zanden (2020), “Maddison style estimates of the evolution of the world economy. A new 2020 update“.
+  #     publisher_source: |
+  #       The Maddison Project Database is based on the work of many researchers that have produced estimates of...
+
+tables:
+  deaths_karlinsky:
+    title: International Completeness of Death Registration 2015-2019 – Karlinsky (2021)
+    description: >
+      Contains all metrics used in the International Completeness of Death Registration 2015-2019 paper.
+    variables:
+      # Life expectancy
+      reg_deaths:
+        title: Registered deaths
+        unit: "deaths"
+        description: >
+          The number of deaths registered in the vital registration system for the country-year. NA denotes no registered deaths data was obtained.
+      expected_wpp:
+        title: Expected number of deaths (WPP 2019)
+        unit: deaths
+        description: >
+          The number of deaths expected to occur for the country-year from World Population Prospects 2019.
+
+      expected_gbd:
+        title: Expected number of deaths (GBD 2019)
+        unit: deaths
+        description: >
+          The number of deaths expected to occur for the country-year from Global Burden of Disease 2019.
+
+      expected_ghe:
+        title: Expected number of deaths (GHE 2019)
+        unit: deaths
+        description: >
+          The number of deaths expected to occur for the country-year from Global Health Estimates 2019.
+
+      expected_deaths:
+        title: Expected number of deaths
+        unit: deaths
+        description: >
+          The mean of expected_wpp, expected_gbd and expected_ghe.
```

Comment:
Given that these descriptions will end up appearing in grapher (where the user does not see the variable short name, but the title) I'd suggest referring to variables by their title.